webapp-django/crashstats/symbols/tests/test_views.py

```diff
@@ -166,6 +161,15 @@ def test_home(self):
             ok_(link['url'] in response.content)
             ok_(link['label'] in response.content)
 
+        # The access should disappar if you cease to be active
+        User.objects.filter(id=user.id).update(is_active=False)
```

Comment:
I think what I'm going to do instead is to re-write the `_login` method so that it returns a user **with** a `last_login` set.